### ISSUE TITLE
Getting error "invalid type: boolean `false`, expected struct Verbosity"

### ISSUE THREAD

**Issue by vipulgupta2048** [lycheeverse/lychee-action#191]
Recently upgraded to latest Lychee action and getting this error when I tried changing verbose value to `info`, I changed value back to false but still getting the same error. 

```
Run /home/runner/work/_actions/lycheeverse/lychee-action/a12cbb3a4921899299dcc58a6136bcdd0d1fbf1c/entrypoint.sh
Error: TOML parse error at line 5, column 11
  |
5 | verbose = false
  |           ^^^^^
invalid type: boolean `false`, expected struct Verbosity
```

Reference: https://github.com/balena-io/docs/actions/runs/4591570995/jobs/8107838446

**Comment by mre** [comment]
Hey Vipul,
this got fixed in https://github.com/lycheeverse/lychee/pull/956, but it's not released yet.
Apologies for the inconvenience.

**Comment by vipulgupta2048** [comment]
No inconvenience at all, happy to test out new versions and report what I see.

**Comment by mre** [comment]
This should be fixed with 1.7.0 now @vipulgupta2048. Let me know if you still have problems and we can reopen. 🙏
